.github/workflows/release.yml

```diff
@@ -21,8 +21,9 @@ jobs:
           # either 'goreleaser' (default) or 'goreleaser-pro'
           distribution: goreleaser-pro
           version: "v2.12.5"
-          args: build --single-target --skip=validate --clean --config=.goreleaser-windows.yml
+          args: release --single-target --skip=validate,publish,blobs--clean --config=.goreleaser-windows.yml
         env:
+          GORELEASER_CURRENT_TAG: ${{ github.ref_name }}
```

Comment:
This workflow runs on tagged release only, this should not be necessary. It might not hurt either but I think we should avoid introducing changes that are not required.